ansible/roles/software/ckan/catalog/ckan-app/tasks/main.yml

```diff
@@ -90,8 +90,9 @@
     - reload apache2
     - reload supervisor
 
-- name: point site_url in /etc/hosts to localhost
-  action: lineinfile dest=/etc/hosts line="127.0.0.1 {{ ckan_site_domain | regex_replace('https?\://','') }}"
+# TODO remove this after production deploy
```

Comment:
Is this a manual step for every deploy, or a note to fix this in code after this particular deploy?

Reply:
It is a note to fix the code after the next deploy.